DB/src/main/java/io/deephaven/db/v2/remote/ConstructSnapshot.java

```diff
@@ -1036,7 +1035,7 @@ public static long callDataSnapshotFunction(@NotNull final LogOutputAppendable l
         int numConcurrentAttempts = 0;
 
         final LivenessManager initialLivenessManager = LivenessScopeStack.peek();
-        while (numConcurrentAttempts < MAX_CONCURRENT_ATTEMPTS && !state.locked()) {
+        while (numConcurrentAttempts < MAX_CONCURRENT_ATTEMPTS) {
```

Comment:
This change doesn't make sense. If we hold the lock, we should fall through straight to the always-successful current-values-only approach. There's nothing correct we could be doing that needs this change.